keanu-python/keanu/sequence.py

```diff
@@ -31,6 +31,9 @@ def add(self, vertex: Vertex, label: Optional[str] = None) -> None:
     def get(self, label: str) -> Vertex:
         return Vertex._from_java_vertex(self.unwrap().get(_VertexLabel(label).unwrap()))
 
+    def get_contents(self) -> Dict[str, Vertex]:
+        return {k.getUnqualifiedName(): Vertex._from_java_vertex(v) for k, v in self.unwrap().getContents().items()}
```

Comment:
I guess we'd only want the unqualified name because the namespace used here is autogenerated?

Reply:
yep - if you were trying to access vertices you'd have to use the label `Sequence_Item_****hash****.label`. The full label can still be obtained from the vertex itself if necessary.